back/config/schemas/settings.schema.json.erb

```diff
@@ -500,6 +500,12 @@
             "type": "integer",
             "title": "OSM Relation ID",
             "description": "The boundary of the municipality/organisation is specified by its Relation ID in OpenStreetMap (OSM). You can find the Relation ID at https://www.openstreetmap.org. Search for the place, click on the correct result and you'll find the Relation ID if the result is a relation (e.g. \"Relation: Knokke-Heist (4569)\"). If there is no relation defined for the place (e.g. Scheveningen), search for the next largest administrative territory (e.g. The Hague)."
+          },
+          "esri_integration": {
+            "type": "boolean",
+            "default": true,
+            "title": "Esri Integration",
+            "description": "Enable Esri integration for maps. In development. DO NOT ENABLE ON COMMERCIAL PLATFORM!"
```

Comment:
> DO NOT ENABLE ON COMMERCIAL PLATFORM!

Should it be `"default": false` then?

Reply:
It could be, but I wanted to make it easy for FE to use while developing without needing to set it each time, for example, the BE is rebuilt and an epic deploy is pushed.

But yes, I need to remember to set the default to `false` before releasing the feature branch.

Reply:
I'll set it to `false` as default. On balance, the extra work to set it is not much, but the approach is 'safer'.